app/views/layouts/_nav.html.haml

```diff
@@ -2,21 +2,27 @@
 - if simplified_nav
   - pages = ['Home']
 - else
-  - pages = ['Timer', 'Announcements', 'Important Info', 'Schedule', 'Challenges', 'Slack']
+  - pages = ['About', 'Schedule', 'Travel', 'FAQ', 'Sponsors']
 
 - api_list = current_page?(:controller => 'pages', :action => 'apilist')
+- live_path = current_page?(:controller => 'pages', :action =>'live')
 - if api_list
   - pages = ['Home']
 
 
 %nav.section.nav
   .nav__links
     - pages.each do |page|
-      - if api_list || simplified_nav
+      - if api_list || simplified_nav 
         %a.nav__link{ href: "/" }= page
       - else
         %a.nav__link{ href: "##{page.downcase}" }= page
+
+
+    %a.nav__link{ href: "https://brickhackv.devpost.com", target: 'blank' }= 'Prizes'
+    %a.nav__link{href: "/live"}= 'Live'
```

Comment:
I think we should comment this out for now until we want people visiting it this weekend